tastytrade/instruments.py

```diff
@@ -124,14 +143,56 @@ def from_dict(cls, json: dict[str, Any]) -> 'Equity':
         snake_json = snakeify(json)
         return cls(**snake_json)
 
+    @classmethod
+    def get_active_equities(
+        self,
+        session: Session,
+        per_page: int = 1000,
+        page_offset: int = 0,
+        lendability: Optional[str] = None,
+    ) -> list['Equity']:
+        url = f'{session.base_url}/instruments/equities/active'
+        equities = []
+        while True:
+            params = {
+                'per-page': per_page,
+                'page-offset': page_offset,
+                'lendability': lendability,
+            }
+            response = requests.get(url, headers=session.headers, params=params)
+            validate_response(response)
+            response_data = response.json()
+            equities.extend(
+                [
+                    Equity(
+                        tick_sizes=[
+                            TickSizes(**ts) for ts in equity.pop('tick-sizes', [])
+                        ],
+                        option_tick_sizes=[
+                            TickSizes(**ots)
+                            for ots in equity.pop('option-tick-sizes', [])
+                        ],
+                        **{k.replace('-', '_'): v for k, v in equity.items()},
+                    )
+                    for equity in response_data['data']['items']
+                ]
+            )
+            total_items = response_data['pagination']['total-items']
+            if page_offset * per_page >= total_items:
+                break
+            else:
+                page_offset += 1
+
+        return equities
+
     @classmethod
     def get_equities(
         cls,
         session: Session,
         symbols: Optional[list[str]] = None,
         lendability: Optional[str] = None,
         is_index: Optional[bool] = None,
-        is_etf: Optional[bool] = None
+        is_etf: Optional[bool] = None,
```

Comment:
another trailing comma to remove

Reply:
I just realized that my linter adds these every time I save...